SparkPost/2web2ui
# AC-1417 Add Segment Scripts to Head

### What Changed
 - Added the Segment.io scripts

### How To Test
 - Check the <head> tag and find the segment script added to the head. Check that the correct key is there (based on value in config files).
 - Ensure logs are being propagated into https://app.segment.com/sparkpost-daniel-chalef/sources/webui_dev/debugger
